src/pkg/cli/compose/validation.go

```diff
@@ -312,6 +312,11 @@ func ValidateProject(project *composeTypes.Project) error {
 	for k := range project.Extensions {
 		term.Warnf("unsupported compose extension: %q", k)
 	}
+
+	if len(project.Volumes) > 0 {
```

Comment:
I think it's more important to warn about them actually being used, ie. the `volumes` field in each service.